src/app/meta-campaign/meta-campaign.component.html

```diff
@@ -2,10 +2,12 @@
   <mat-card-header>
     <div mat-card-avatar class="childhood-trust-icon"></div>
     <mat-card-title>{{ campaign.title }}</mat-card-title>
+    <mat-card-subtitle *ngIf="campaign.status == 'Preview'">Opens in {{ campaign.startDate | timeLeft }}</mat-card-subtitle>
```

Comment:
@NoelLH This looks good. Can we add logic to say `IF campaign.startDate == Today` then show `Opens today!` as this looks a bit weird:

![image](https://user-images.githubusercontent.com/47176130/63875310-1c289600-c9bb-11e9-84c0-6e2981895b69.png)

Reply:
@tbg-jasonfung as noted on the Jira task I think we should leave refinements + exact presentation of any countdown until we have both more time and some UX input